docs/cli-reference/dfx-nns.md

```diff
@@ -1,158 +1,2 @@
-
-* The NNS canisters need to run on a system subnet.
-* Some canisters are compiled to run on only very specific canister IDs and hostname/port pairs.
-
-
-In addition, the local dfx server needs to be clean:
-
-``` bash
-$ dfx start --clean --background
-$ dfx nns install
-```
-
-This is because NNS canisters need to be installed before any others.
-
-
-### Examples
-
-#### Example: Making API calls to the local NNS.
-
-``` bash
-$ dfx stop
-$ dfx start --clean --background
-$ dfx nns install
-$ dfx nns import
-$ dfx canister call --network ic nns-governance get_pending_proposals '()'
-```
-
-You can view the API calls that can be made for each NNS canister by looking at the interface definition files installed by `dfx nns import` in `candid/*.did`.  The API methods are in the `service` section, which is usually located at the end of a `.did` file.  It is easiest to start experimenting with methods that take no arguments.
-
-#### Example: Accessing ICP on the command line
-Two accounts in the local ledger is initialized with ICP that can be used for testing.  One uses a secp256k1 key, which is convenient for command line usage, another uses an ed25519 key, which is more convenient in web applications.
-
-
-
-To use ICP on the command line:
-* Start dfx and install the NNS, as described in [`install`](#_dfx_nns_install).
-* Put this secret key into a file called `ident-1.pem`:
-``` bash
-$ cat <<EOF >ident-1.pem
------BEGIN EC PRIVATE KEY-----
-MHQCAQEEICJxApEbuZznKFpV+VKACRK30i6+7u5Z13/DOl18cIC+oAcGBSuBBAAK
-oUQDQgAEPas6Iag4TUx+Uop+3NhE6s3FlayFtbwdhRVjvOar0kPTfE/N8N6btRnd
-74ly5xXEBNSXiENyxhEuzOZrIWMCNQ==
------END EC PRIVATE KEY-----
-EOF
-```
-* Check the key: (optional)
-```
-$ openssl ec -in ident-1.pem -noout -text
-```
-* Create an identity with that secret key:
-``` bash
-$ dfx identity import ident-1 ident-1.pem
-```
-* Now you can use the (toy) funds:
-``` bash
-$ dfx ledger balance
-```
-
-To use ICP in an existing web application:
-* Install the [@dfinity/agent npm module](https://www.npmjs.com/package/@dfinity/agent).
-* Create an identity with this key pair:
-```
-  const publicKey = "Uu8wv55BKmk9ZErr6OIt5XR1kpEGXcOSOC1OYzrAwuk=";
-  const privateKey =
-    "N3HB8Hh2PrWqhWH2Qqgr1vbU9T3gb1zgdBD8ZOdlQnVS7zC/nkEqaT1kSuvo4i3ldHWSkQZdw5I4LU5jOsDC6Q==";
-  const identity = Ed25519KeyIdentity.fromKeyPair(
-    base64ToUInt8Array(publicKey),
-    base64ToUInt8Array(privateKey)
-  );
-
-  // If using node:
-  const base64ToUInt8Array = (base64String: string): Uint8Array => {
-    return Buffer.from(base64String, 'base64')
-  };
-  // If in a browser:
-  const base64ToUInt8Array = (base64String: string): Uint8Array => {
-    return Uint8Array.from(window.atob(base64String), (c) => c.charCodeAt(0));
-  };
-```
-* That identity can now make API calls, including sending ICP.
+# dfx nns
+> **NOTE**: The nns command is no longer part of dfx, and has now been turned into dfx extension. In order to obtain it, please run `dfx extension install nns`
```

Comment:
Have we moved these docs somewhere else (where they appear on the portal?). If not, we should leave the docs here (along with this note)

Reply:
Not yet; made a ticket for this https://dfinity.atlassian.net/browse/SDK-1174 

These docs were turned into readme pages https://github.com/dfinity/dfx-extensions/blob/main/extensions/sns/README.md & https://github.com/dfinity/dfx-extensions/blob/main/extensions/nns/README.md

@ericswanson-dfinity @sesi200 @jessiemongeon1 : Is integrating these readmes into dfinity/portal a good idea? if yes, how would you go about doing it

Reply:
Let's not reduce the documentation available on the portal.  We should be able to set up docs on the dfx-extensions repo that are accessible from the portal, just as they are available here.

In other words: make things a little bit better, not a little bit worse.  That may mean leaving these docs in place here until they are accessible from the portal on dfx-extensions.